plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftMetastoreModule.java

```diff
@@ -28,12 +28,17 @@
 import jakarta.annotation.PreDestroy;
 
 import java.util.concurrent.ExecutorService;
+import java.util.function.Supplier;
```

Comment:
Please update `TestHivePlugin`.

Reply:
Moved the tests in `TestHiveConnectorFactory` to `TestHivePlugin` as discussed over slack.